docs/protocol/concepts/accounts.md

```diff
@@ -4,22 +4,91 @@ sidebar_position: 1
 
 # Accounts
 
-This document describes the in-built accounts system of Evmos.
+Crypto Wallets (or Accounts) can be created
+and represented in unique ways on differnt blockchains.
+For developers who interface with account types on Evmos,
+e.g. during Wallet integration on their dApp frontend,
+it is therefore important to understand that accounts on Evnos are implemented
+to be compatible with Ethereum type addresses.
 
 ## Prerequisite Readings
 
 - [Cosmos SDK Accounts](https://docs.cosmos.network/main/basics/accounts.html)
 - [Ethereum Accounts](https://ethereum.org/en/whitepaper/#ethereum-accounts)
 
+## Creating Accounts
+
+To create one account you can either
+create a private key, a keystore file (a private key protected by a password),
+or a mnemonic phrase (a string of words that can access multiple private keys).
+
+Aside from having different security features,
+the biggest difference between each of these is
+that a private key or keystore file only creates one account.
+Creating a mnemonic phrase gives you control of many accounts,
+all accessible with that same phrase.
+
+Cosmos blockchains, like Evmos, support creating accounts with mnemonic phrases,
+otherwise known as a [hierarchical deterministic key generation](https://github.com/confio/cosmos-hd-key-derivation-spec) (HD keys).
```

Comment:
```suggestion
otherwise known as [hierarchical deterministic key generation](https://github.com/confio/cosmos-hd-key-derivation-spec) (HD keys).
```